internal/coremsgs/en_config_descriptions.go

```diff
@@ -20,301 +20,265 @@ import (
+	ConfigGlobalRetryInitialDelay = ffc("config.global.retry.initialDelay", "The initial retry delay", timeDurationType)
+	ConfigGlobalRetryMaxDelay     = ffc("config.global.retry.maxDelay", "The maximum retry delay", timeDurationType)
+	ConfigGlobalRetryMaxAttempts  = ffc("config.global.retry.maxAttempts", "The maximum number of times to retry", intType)
+	ConfigGlobalRetryCount        = ffc("config.global.retry.count", "The maximum number of times to retry", intType)
+	ConfigGlobalInitWaitTime      = ffc("config.global.retry.initWaitTime", "The initial retry delay", timeDurationType)
+	ConfigGlobalMaxWaitTime       = ffc("config.global.retry.maxWaitTime", "The maximum retry delay", timeDurationType)
+
+	ConfigGlobalUsername = ffc("config.global.auth.username", "Username", stringType)
+	ConfigGlobalPassword = ffc("config.global.auth.password", "Password", stringType)
+
+	ConfigGlobalSize = ffc("config.global.cache.size", "The size of the cache", byteSizeType)
+	ConfigGlobalTTL  = ffc("config.global.cache.ttl", "The time to live (TTL) for the cache", timeDurationType)
+
+	ConfigGlobaltWsHeartbeatInterval     = ffc("config.global.ws.heartbeatInterval", "The amount of time to wait between heartbeat signals on the WebSocket connection", timeDurationType)
+	ConfigGlobalWsInitialConnectAttempts = ffc("config.global.ws.initialConnectAttempts", "The number of attempts FireFly will make to connect to the WebSocket when starting up, before failing", intType)
+	ConfigGlobalWsPath                   = ffc("config.global.ws.path", "The WebSocket sever URL to which FireFly should connect", "WebSocket URL "+stringType)
+	ConfigGlobalWsReadBufferSize         = ffc("config.global.ws.readBufferSize", "The size in bytes of the read buffer for the WebSocket connection", byteSizeType)
+	ConfigGlobalWsWriteBufferSize        = ffc("config.global.ws.writeBufferSize", "The size in bytes of the write buffer for the WebSocket connection", byteSizeType)
+
+	ConfigGlobalTLSCaFile           = ffc("config.global.tls.caFile", "The path to the CA file for the admin API", stringType)
+	ConfigGlobalTLSCertFile         = ffc("config.global.tls.certFile", "The path to the certificate file for the admin API", stringType)
+	ConfigGlobalTLSClientAuth       = ffc("config.global.tls.clientAuth", "Enables or disables client auth for the admin API", stringType)
+	ConfigGlobalTLSEnabled          = ffc("config.global.tls.enabled", "Enables or disables TLS on the admin API", booleanType)
+	ConfigGlobalTLSKeyFile          = ffc("config.global.tls.keyFile", "The path to the private key file for the admin API", stringType)
+	ConfigGlobalTLSHandshakeTimeout = ffc("config.global.tlsHandshakeTimeout", "The maximum amount of time to wait for a successful TLS handshake", timeDurationType)
+
+	ConfigGlobalBodyTemplate          = ffc("config.global.bodyTemplate", "The body go template string to use when making HTTP requests", "Go template "+stringType)
```

Comment:
Would be nice to make go template a type and link off to the go docs - non go people might need some help with the templating `{{ .Stuff }}` syntax